src/main/java/com/epam/task/third/service/TetrahedronSorting.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.task.third.service;
+
+import com.epam.task.third.entities.Tetrahedron;
+
+import java.util.List;
+
+public class TetrahedronSorting  {
```

Comment:
no need in this class, be object oriedted: different comarator implementations + Factory